packages/f/ffmpeg.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "ffmpeg",
+  "filename": "ffmpeg.min.js",
+  "description": "A pure Webassembly / Javascript port of FFmpeg.",
+  "keywords": [
+    "ffmpeg"
+  ],
+  "license": "MIT",
+  "autoupdate": {
+    "source": "npm",
+    "target": "@ffmpeg/ffmpeg",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "**/*.js",
+          "**/*.js.map"
```

Comment:
👋 We try to avoid using globstars in configs as they can lead to including files that weren't intended to be on the CDN.

Looking at the matched contents, I suspect you can just use this here:

```suggestion
          "*.js?(.map)"
```

Reply:
In `@ffmpeg/ffmpeg@0.12.0-alpha.1`, they changed the way they packaged it, which caused the file paths to change.

_After:_

![](https://user-images.githubusercontent.com/47095648/221442742-1a5582a4-acc1-416c-a932-cacec0528f4e.png)

_Before:_

![](https://user-images.githubusercontent.com/47095648/221442756-3aad67c4-b967-4531-a5fa-c79afef245df.png)

Reply:
In that case, I would recommend using `{,umd/,esm/}*.js`

Reply:
Okay I've updated it just now.